lib/main.dart

```diff
@@ -106,43 +153,10 @@ class _MyHomePageState extends State<MyHomePage> {
                 ],
               ),
             ),
-            const Divider(height: 50),
-            Row(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                FittedBox(
-                  fit: BoxFit.fitWidth,
-                  child: Container(
-                    margin: const EdgeInsets.only(right: 50),
-                    child: const Text("1",
-                        style: TextStyle(fontSize: 50.0, color: Colors.grey)),
-                  ),
-                ),
-                Expanded(
-                  child: Column(
-                    children: [
-                      const Align(
-                        alignment: Alignment.centerLeft,
-                        child: Text("Phone Number", style: primaryColor),
-                      ),
-                      const TextField(
-                        decoration: InputDecoration(
-                          hintText: '7019316063',
-                        ),
-                      ),
-                      TextButton(
-                        style: TextButton.styleFrom(
-                            backgroundColor: Colors.orange),
-                        onPressed: null,
-                        child: const Text('Get OTP',
-                            style:
-                                TextStyle(color: Colors.white, fontSize: 10)),
-                      )
-                    ],
-                  ),
-                ),
-              ],
-            ),
+            homepageForm("1", "Phone Number", "Get OTP", "7019316063"),
```

Comment:
Just a suggestion any hardcoded strings we can actually create a file and store them.
This will help at later point when we will be looking for translation, usage at multiple places.

P.S : This can be done at later point of time.